pkgs/top-level/all-packages.nix

```diff
@@ -15910,6 +15910,8 @@ with pkgs;
 
   thinkingRock = callPackage ../applications/misc/thinking-rock { };
 
+  nylas-mail = callPackage ../applications/networking/mailreaders/nylas-mail { };
```

Comment:
Not sure: it could remind that it is a binary patching and maybe incentivise someone to do a source build (and adding a source build would not be confusing in this case)